mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -3156,77 +3201,6 @@ def test_accept_check(self):
             self.assertContains(response, 'Credit accepted')
 
-    def test_invalid_form(self):
-        """
-        Test that if a rejected check is accepted, a validation error is displayed.
-        """
-        check_id = 1
-        response_len = 4
-        with responses.RequestsMock() as rsps:
-            self.login(rsps=rsps)
-            rsps.add(
-                rsps.GET,
-                api_url(f'/security/checks/{check_id}/'),
-                json=self.SENDER_CHECK_REJECTED
-            )
-            rsps.add(
-                rsps.GET,
-                urljoin(
-                    settings.API_URL,
-                    '/senders/{sender_profile_id}/credits/?{querystring}'.format(
-                        sender_profile_id=self.sender_id,
-                        querystring=urlencode([
-                            ('limit', 500),
-                            ('offset', 0),
-                            ('exclude_credit__in', self.credit_id),
-                            ('security_check__isnull', False),
-                            ('only_completed', False),
-                            ('security_check__actioned_by__isnull', False),
-                            ('include_checks', True)
-                        ])
-                    ),
-                    trailing_slash=False
-                ),
-                json={
-                    'count': response_len,
-                    'results': list(self._get_sender_credit_list(response_len))
-                }
-            )
-            rsps.add(
-                rsps.GET,
-                urljoin(
-                    settings.API_URL,
-                    '/prisoners/{prisoner_profile_id}/credits/?{querystring}'.format(
-                        prisoner_profile_id=self.prisoner_id,
-                        querystring=urlencode([
-                            ('limit', 500),
-                            ('offset', 0),
-                            ('exclude_credit__in', ','.join(map(str, ([self.credit_id] + list(range(response_len)))))),
-                            ('security_check__isnull', False),
-                            ('only_completed', False),
-                            ('security_check__actioned_by__isnull', False),
-                            ('include_checks', True)
-                        ])
-                    ),
-                    trailing_slash=False
-                ),
-                json={
-                    'count': response_len,
-                    'results': list(self._get_prisoner_credit_list(response_len))
-                }
-            )
-
-            url = reverse('security:resolve_check', kwargs={'check_id': check_id})
-            response = self.client.post(
-                url,
-                data={
-                    'fiu_action': 'accept',
-                },
-                follow=True
-            )
-
-            self.assertContains(response, 'You cannot action this credit')
-
```

Comment:
Is this test definitely redundant? 

Is the possible case where 2 users have the form open at the same time and then one actions the check before the other something that would still trigger this error occurring (i.e has the code doing the outcome of this test also been removed?)

Reply:
it's redundant in that there is a line-by-line identical test already: `test_invalid_if_check_not_in_pending`. previously, these were on two separate views so made sense, but now that it's one form, they do exactly the same thing. i.e. `test_invalid_if_check_not_in_pending` already tests for the situation you mention

Reply:
Ok, cool, missed the duplicate test in the PR but that's cool then